extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -72,6 +72,7 @@ public function getContext( Title $title, $skinName ) {
 					'wikiLanguage' => $langCode,
 					'wikiVertical' => $newWikiVertical,
 					'newWikiCategories' => $this->getNewWikiCategories( $wikiFactoryHub, $wg->CityId ),
+					'hasPI' => !empty( \Wikia::getProps( $title->getArticleID(), PortableInfoboxDataService::INFOBOXES_PROPERTY_NAME ) )
```

Comment:
It was a request from Ad Eng to keep it short. I personally like the verbosity better.